inputfiles/addedTypes.json

```diff
@@ -2239,7 +2239,7 @@
                         "cloneNode": {
                             "name": "cloneNode",
                             "override-signatures": [
-                                "cloneNode(deep?: boolean): HTMLDialogElement"
+                                "cloneNode(deep?: boolean): HTMLVideoElement"
```

Comment:
Not sure where this one came from probably from conflict.
Thanks for fixing this one.